x/launch/keeper/msg_server_create_chain_test.go

```diff
@@ -29,22 +35,27 @@ func TestMsgCreateChain(t *testing.T) {
 	}{
 		{
 			name:          "valid message",
-			msg:           sample.MsgCreateChain(coordAddress, ""),
+			msg:           sample.MsgCreateChain(coordAddress, "", false, 0),
 			wantedChainID: 0,
 		},
 		{
 			name:          "creates a unique chain ID",
-			msg:           sample.MsgCreateChain(coordAddress, ""),
+			msg:           sample.MsgCreateChain(coordAddress, "", false, 0),
 			wantedChainID: 1,
 		},
 		{
 			name:          "valid message with genesis url",
-			msg:           sample.MsgCreateChain(coordAddress, "foo.com"),
+			msg:           sample.MsgCreateChain(coordAddress, "foo.com", false, 0),
 			wantedChainID: 2,
 		},
+		{
```

Comment:
We can have two additional test cases:
- The campaign doesn't exist
- The coordinator of the campaign and the chain doesn't match